NorthstarDLL/core/hooks.cpp

```diff
@@ -459,4 +472,6 @@ void InstallInitialHooks()
 		spdlog::error("MH_Initialize (minhook initialization) failed");
 
 	AUTOHOOK_DISPATCH()
+
+	disable_igo = strstr(GetCommandLineA(), "-disable_igo");
```

Comment:
as discussed on discord a bit, I'm not a fan of `-disable_igo`.
An alternative could be `-nogameoverlay` or something?

Anyway yeah use this comment as a discussion point for the name or something i guess